sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorConnection.java

```diff
@@ -258,7 +257,7 @@ public int getMaxFrameSize() {
                         connectionId, sessionName), handler.getErrorContext())));
 
             return activeSession.thenReturn(sessionSubscription.getSession());
-        }).or(onClosedError("Could not create session: " + sessionName));
+        });
```

Comment:
Is there any specific  error scenario which it was covering and we should test that ?